### PR TITLE
add: reorganize & use styled components

### LINKED ISSUE #14: Add Loader 🌀
We need to add a loader when getting data from Github.

### LINKED ISSUE #15: Add React Helmet ⛑for SEO
As it's a blog system we need SEO and for that, we need to use React Helmet for SEO. 
If there some other thing we can use for SEO, please suggest

### PR DESCRIPTION
**Pull request by jvm-odoo**
This is a big refactoring which aims to improve the code readability & customization with styled-components.

- It adds header customization in config.js.
- All deleted files are just renamed or moved (except the CSS which is now styled components)
- It uses GraphQL hooks who provides a `loading` state. So I made the loader too.
- It adds react-helmet

closes #14
closes #15

### CODE REVIEW

**Comment by jvm-odoo** [comment]
Job done @saadpasta ! Let me kow what you think about that

**Comment by saadpasta** [comment]
@jvm-odoo Great Work. I look into it

**Comment by jvm-odoo** [comment]
Thanks. I pushed a fix to allow background gradients in the header

File: src/Application.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { ApolloProvider } from '@apollo/react-hooks';
+import { Helmet } from "react-helmet";
+
+import { config } from './config'
+import { client } from './Utils/apollo';
+import Router from './Router';
+
+const Application = () => (
+  <>
```
Comment (saadpasta) [inline src/Application.js]: Why you used empty bracket <> ?

Comment (saadpasta) [reply src/Application.js]: You can use React Fragment.

Comment (jvm-odoo) [reply src/Application.js]: I am glad that you learn new things. This is why I contribute to open-source projects.
<> is a shortcut for React Fragment (and do not need importing something)

Comment (saadpasta) [reply src/Application.js]: Oh Great 😀

**Pull request merged**